engine/src/vegastrike.h

```diff
@@ -75,14 +75,14 @@ extern float AUDIO_ATOM;
 #if !defined (NO_GFX)
 #   if defined(__APPLE__) && defined(__MACH__)
 #       define GL_GLEXT_FUNCTION_POINTERS 1
-#       include <gl.h>
-#       include <glu.h>
-#       include <glut.h>
+#       include <GL/gl.h>
+#       include <GL/glu.h>
+#       include <GL/glut.h>
```

Comment:
Um. I know it's non-standard to include these headers without the directory prefix, but we're doing it this way on purpose. It's been discussed previously; I just don't remember where off the top of my head.

This might be part of the build problems, as well.